lib/xcode_summary/plugin.rb

```diff
@@ -20,11 +20,16 @@ module Danger
   #
   class DangerXcodeSummary < Plugin
     attr_writer :project_root
+    attr_writer :ignored_files
 
     def project_root
       @project_root || Dir.pwd
     end
 
+    def ignored_files
+      [@ignored_files].flatten.compact
```

Comment:
this makes sure we can support arrays and single strings
it also returns `[]` when `@ignored_files.nil?`